ergotree-interpreter/src/eval/xor_of.rs

```diff
@@ -0,0 +1,40 @@
+use ergotree_ir::mir::constant::TryExtractInto;
+use ergotree_ir::mir::value::Value;
+
+use crate::eval::env::Env;
+use crate::eval::EvalContext;
+use crate::eval::EvalError;
+use crate::eval::Evaluable;
+use ergotree_ir::mir::xor_of::XorOf;
+
+impl Evaluable for XorOf {
+    fn eval(&self, env: &Env, ctx: &mut EvalContext) -> Result<Value, EvalError> {
+        let input_v = self.input.eval(env, ctx)?;
+        let input_v_bools = input_v.try_extract_into::<Vec<bool>>()?;
+        Ok((input_v_bools.into_iter().filter(|x| *x).count() & 1 == 1).into())
```

Comment:
Frankly, I don't find it very readable. Although I'm biased here, since I was the one who botched the Scala implementation - https://github.com/ScorexFoundation/sigmastate-interpreter/issues/640 and now I'm blowing on the water. So let's leave it as is. :)